src/widgets/projectInfo/types.ts

```diff
@@ -0,0 +1,28 @@
+import {type DashboardWidget} from '../../types'
+
+export interface ProjectInfoProps {
+  __experimental_before?: DashboardWidget[]
+  data: ProjectData[]
+}
+
+export interface App {
+  title: string
+  rows?: App[]
+  value?: string | {error: string}
+}
+
+export interface ProjectData {
+  title: string
+  category?: string
+}
+
+export interface UserApplication {
+  id: string
+  projectId: string
+  title: string | null
+  type: string
+  urlType: 'internal' | 'external'
+  appHost: 'espen'
```

Comment:
Should this be string?

```suggestion
  appHost: string
```

Reply:
lol oops